DESCRIPTION

```diff
@@ -4,7 +4,7 @@ Title: Interface to 'TensorFlow' Datasets
 Version: 2.17.0.9000
 Authors@R: c(
   person("Tomasz", "Kalinowski", role = c("ctb", "cph", "cre"),
-         email = "tomasz@posit.co"),
+         email = "daniel@posit.co"),
```

Comment:
Is this intentional?

Reply:
Oh, sorry, I changed to get a cran check into my email.

Reply:
Thanks!